### ISSUE TITLE
Suggestion on plain requests

### ISSUE THREAD

**Issue by rui-ribeiro** [andyjeffries/flexirest#37]
Hi mate

I have a question/suggestion.

So, as you know, to make a plain request I have to do something like:

```ruby
.. 
Person._plain_request('http://api.example.com/v1/people', :post, {id:1234,name:"John"})
```

Why I cant do it like this?

```ruby
.. 
Person.create(id:1234,name:"John", plain: true)
```

Also do you think to add multipart sometime?

Thanks :)

**Comment by andyjeffries** [comment]
Hey mate.

I don't get what the `plain: true` would do in that case.  The difference between a mapped request and a plain request is that you don't know the URL for a plain request ahead of time, so you have to specify it manually.  If you know the URL for the create call, why would you need to pass `plain: true` in to it?

Never thought of multipart form data because I've never heard of an API needing it (and Flexirest is an API client rather than a general HTTP client).  Do you know of a public API needing multipart?

**Comment by rui-ribeiro** [comment]
Sometimes you want to make a plain request because is faster because if you do a normal FlexiRest request a object will be created and if the response is big will take time. 

So, I know the url but I don´t want to map the object

A public API no, but the API (private) I´m working needs it.

**Comment by andyjeffries** [comment]
So, I went to write that `plain: true` type of syntax and found it was already in there! So I added a test for it and added it to the documentation - https://github.com/andyjeffries/flexirest#plain-requests and then released it as v1.3.13.

Also, the multipart thing can be handled using a Faraday configuration block (https://github.com/andyjeffries/flexirest#faraday-configuration) and Faraday's built-in `:multipart` middleware (https://github.com/lostisland/faraday#advanced-middleware-usage).

Hope this helps.
